source/patterns/@aws-solutions-constructs/core/lib/utils.ts

```diff
@@ -176,4 +177,20 @@ export function consolidateProps(defaultProps: object, clientProps?: object, con
   }
 
   return result;
-}
\ No newline at end of file
+}
+
+/**
+ * Generates a name unique to this location in this stack with this stackname. Truncates to under 64 characters if needed.
+ * (will allow 2 copies of the stack with different stack names, but will collide if both stacks have the same name)
+ *
+ * @param scope the construct within to create the name
+ * @param resourceId an id for the construct about to be created under scope (empty string if name is for scoep)
+ * @returns a unique name
+ */
+export function generateName(scope: Construct, resourceId: string = ""): string {
+  const name = /*cdk.Stack.of(scope).region +*/ resourceId + cdk.Names.uniqueId(scope);
```

Comment:
is the commented out `cdk.Stack.of(scope).region` intentional? If so, can we add a comment as to what the intent of it is?